### DISCUSSION TITLE
domino/14.0.0/admin/conf_dominopasskeyauth

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#95] [Q&A]
# domino/14.0.0/admin/conf_dominopasskeyauth

Learn how to enable passkey authentication on a Domino server or internet site for improved security.

https://help.hcl-software.com/domino/14.0.0/admin/conf_dominopasskeyauth.html

**Comment by kbmsg** [comment]
Please elaborate on the section marked "What to do next" with the following:
What is the user's experience?
You don't even show what the basic correct domcfg login looks like.
How does a user set this up on their end?
How does a user know if Passkeys are even enabled?

**Comment by kbmsg** [comment]
Please create a section at the top that provides the guidelines/requirements for Passkey support.
This is missing and badly needed.
Windows 10 companies need to find a 3rd party solution since Windows does not handle Passkeys.
Windows 11 kind of works, but needs to be defined for the users via policies most likely.
Windows Hello which is the MS Passkey option is different when used by an Enterprise, than by a non Enterprise device and that needs to be a consideration.
Laptops may or may not have facial/finger identification and that needs to be considered.
Apple products have it built in by default, but few organizations rely on Apple products.
A 3rd party app may be required, possibly recommend 2-3.
Traveler(Verse for Mobile clients) DOES NOT WORK with Passkeys. So companies needd to stay with TOTP for now.
I am sure there are other things to think about.

**Comment by kbmsg** [comment]
It would be helpful if there was a chart to compare TOTP and Passkeys so customers could understand what HCL Domino products support which MFA and when is one preferred over the other.
Or, even, if there is an End of Life coming for TOTP inside Domino
